website/static/js/project-organizer.js

```diff
@@ -364,10 +375,14 @@ var tbOptions = {
                     }
                     tb.filter(event);
                 },
-                onchange: m.withAttr('value', tb.filterText),
+                onchange: function() {
+                    m.withAttr('value', tb.filterText);
```

Comment:
`m.withAttr` is helper for returning callback functions.

```js
onchange: m.withAttr('value', someProp)
```

is equivalent to

```js
onchange: function(e) { someProp(e.value }
```

m.withAttr returns a function (but does not call it), which is why this line looked strange to me. However, I'm now wondering if this line is even necessary, given that `filterText` is already updated in an `onkeyup` handler.